README.md

```diff
@@ -372,6 +372,15 @@ If you have received the package and only want to build and run the docker conta
         concent-signing-service:latest
     ```
 
+### Generate Signing Service keys
+To generate public and private keys for signing-service use the `generate-ecc-key-pair.sh` script.
```

Comment:
Please reword:
```
### Generating a key pair
Signing Service needs a key pair to secure its communication with Concent.
These are the same keys that are used for signing messages generated by golem-messages.
They're just base64-encoded so that they can be easily passed on the command line or in shell variables.

The public key needs to be known to Concent.
The private one must not - this is the whole point of having a separate service for signing.

When setting up a new instance of the service, you can easily generate a new pair using the `generate-ecc-key-pair.sh` script included in the Docker image:
```